libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -283,6 +290,16 @@ void AP_Camera::update()
         return;
     }
 
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane) || APM_BUILD_TYPE(APM_BUILD_APMrover2)
+    if (_control_mode != 10 && _auto_only == true) {
```

Comment:
better to use != 0 than == true to since a param like this may get extended later and the true/false check isn't changed.

Reply:
Did you try using AUTO instead of 10 and 3 for control_mode? I think the defines should work here.. Having explicit numbers are a bad idea

Reply:
Yes, I did try AUTO, it's not visible to AP_Camera, and I the defines.h files for Copter/Plane are different too, so I cannot just import one,
Will change to true within minutes :)
- thank you.

Reply:
Changed to != 0